apps/client/tsconfig.json

```diff
@@ -12,7 +12,7 @@
       "vitest/globals"
     ],
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
question: can you please leave a comment to explain the changes in this file?

Reply:
Clarified in https://github.com/cpvalente/ontime/pull/1438/commits/5b3e6000e61154f842eb3379c5ac455585ab1797